### ISSUE TITLE
Loss function in CNNClassifier

### ISSUE THREAD

**Issue by vanyalzr** [sktime/sktime-dl#73]
Hi, 

Thanks for the great work! I was looking at the implementation of `CNNClassifier` and noticed that the loss function used for this model appears to be [MSE](https://github.com/sktime/sktime-dl/blob/feb42cf327371531c56c839c9dc24e786c0a34c4/sktime_dl/deeplearning/cnn/_classifier.py#L93). Is there any specific reason for this choice of the loss function? Other models seem to use cross-entropy which should potentially work better for classification problems.

**Comment by mloning** [comment]
Hi @vanyalzr, thanks for raising the issue. I'm not sure, @James-Large may be able to comment on this. 

A simple solution would be to expose the choice of loss function in the constructor to allow users to set whatever loss they prefer.

**Comment by TonyBagnall** [comment]
closing this as I think it solved, feel free to raise again after restructure
